stock_barcodes/tests/test_stock_barcodes.py

```diff
@@ -139,3 +139,7 @@ def test_barcode_context_action(self):
                     "context": "{'search_default_code': 'incoming'}  ",
                 }
             )
+
+    def test_compute_show_form_scan(self):
```

Comment:
This test is not useful and can be tricky, as you are calling directly a compute method. You should reproduce real flows where the compute is called naturally and you get the expected result.